### ISSUE TITLE
ResNet-RS block/layer

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#36]

**Comment by bhack** [comment]
It could be extrapolated/refactored from https://github.com/tensorflow/models/blob/master/official/vision/beta/MODEL_GARDEN.md#resnet-rs-models-trained-with-various-settings

**Comment by sebastian-sz** [comment]
Currently, there are two implementations available:

1. Linked in the paper from [tensorflow/tpu](https://github.com/tensorflow/tpu/blob/cb119a8607ee9bb70542867579dc9b6e69a85c5a/models/official/resnet/resnet_model.py#L385) repository.
2. Independent implementation from [Model Garden](https://github.com/tensorflow/models/blob/master/official/vision/beta/MODEL_GARDEN.md#resnet-rs-models-trained-with-various-settings)

The first one uses fixed padding for conv2d's and relu activation's, while the second one uses regular convs and swish. Which one would be preferred in this repository?

**Comment by LukeWood** [comment]
We aren't exposing blocks as an API
